code/__DEFINES/blood.dm

```diff
@@ -0,0 +1,15 @@
+/// Checks if an object is covered in blood
+#define HAS_BLOOD_DNA(thing) (length(thing.GetComponent(/datum/component/forensics)?.blood_DNA))
+
+//Bloody shoes/footprints
+#define BLOODY_FOOTPRINT_BASE_ALPHA 20 /// Minimum alpha of footprints
+#define BLOOD_AMOUNT_PER_DECAL      50 /// How much blood a regular blood splatter contains
+#define BLOOD_ITEM_MAX              200 /// How much blood an item can have stuck on it
+#define BLOOD_POOL_MAX              300 /// How much blood a blood decal can contain
+#define BLOOD_FOOTPRINTS_MIN        5 /// How much blood a footprint need to at least contain
+
+//Bloody shoe blood states
+#define BLOOD_STATE_HUMAN "blood" /// Red blood
+#define BLOOD_STATE_XENO "xeno" /// Green xeno blood
+#define BLOOD_STATE_OIL "oil" /// Black robot oil
+#define BLOOD_STATE_NOT_BLOODY "no blood whatsoever" /// No blood is present 
```

Comment:
same as above, autodoc comments need to be above the define.

Reply:
https://github.com/tgstation/tgstation/pull/63879/commits/4097b0215c476f4356e412248b0d4a9e76209363